static/templates/login_to_access.hbs

```diff
@@ -0,0 +1,31 @@
+<div id="login_to_access_modal" class="modal modal-bg fade new-style" tabindex="-1" role="dialog" aria-labelledby="login_to_access_modal" aria-hidden="true">
+    <div class="modal-header">
+
+        <div class="color_animated_button go_back_button">
+            <i class="fa fa-arrow-left"></i>
+            <span class="login_to_access_label"><b>{{t "Login required" }}</b></span>
+        </div>
+    </div>
+    <div class="modal-body">
+        <p>
+            {{#tr}}
+                Since you are not logged in, you can only view messages in
+                <z-link>web public streams</z-link>.
+                {{#*inline "z-link"}}<a target="_blank" href="https://zulipchat.com/help/stream-permissions">{{> @partial-block}}</a>{{/inline}}
+            {{/tr}}
+        </p>
+        <div class="topic_edit_spinner"></div>
+    </div>
+    <div class="modal-footer">
+        <div class="login_to_access_footer">
+            <div class="singup_button color_animated_button">
+                <i class="fa fa-pencil-square-o"></i>
+                <span>{{t "Sign up" }}</span>
+            </div>
+            <div class="login_button color_animated_button">
+                <i class="fa fa-sign-in"></i>
+                <span>{{t "Log in" }}</span>
+            </div>
```

Comment:
These should be HTML <button> elements, not spans, right?  I think that results in ARIA stuff being correct, etc.

Also, `singup` should be `signup` and I don't think that button is hooked up to do anything.

Reply:
Ah! my bad. `span` is fine here because we want the text to be inline with the icon.